microsoft-iis-mixin/dashboards/microsoft-iis-applications.libsonnet

```
@@ -0,0 +1,1291 @@
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'reqps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local requestErrorsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(app, instance, job, status_code) (rate(windows_iis_worker_request_errors_total{job=~"$job", instance=~"$instance", app=~"$application"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - {{app}} - {{status_code}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Request errors',
+  description: 'Requests that have resulted in errors for an IIS application.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'reqps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local websocketConnectionAttemptsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(app, instance, job) (increase(windows_iis_worker_websocket_connection_attempts_total{job=~"$job", instance=~"$instance", app=~"$application"}[$__interval:]))',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - {{app}}',
```

dashboard-to-mixin *doesn't* add min step, needs to be done manually after:

```suggestion
      interval: '1m',
```

making this comment once, but needs to be updated everywhere (16 panels total)